solutions/search/apis-and-tools.md

```diff
@@ -74,14 +74,18 @@ The accompanying [GitHub repository](https://www.github.com/elastic/elasticsearc
 
 ## Generative AI tools
 
+The following generative AI tools are available as part of the {{es}} solution and serverless project type:
```

Comment:
Consider some sort of value prop here - these are value adds that let you do xyz